### PR TITLE
Remove node 14 and add arm64-linux.

### PR DESCRIPTION
**Pull request by cjyar**
This is required for ironcorelabs/cmk-config-broker#1030.

### CODE REVIEW

**Comment by leeroy-travis** [comment]
Commit SHA:359e3adf7b3810148ef579482c1afc15810c268e
No changes to code coverage between the base branch and the head branch

**Comment by leeroy-travis** [comment]
Commit SHA:11e49756063e93ab26c266dd739f03cd8e0a30e1
No changes to code coverage between the base branch and the head branch

**Comment by leeroy-travis** [comment]
Commit SHA:a21e8a8966f10e1d94b4cfa9dff355da6b3221b5
No changes to code coverage between the base branch and the head branch

**Comment by leeroy-travis** [comment]
Commit SHA:bebbdf636ff034ee3aa5a696cbfc60bdca0b3943
No changes to code coverage between the base branch and the head branch

**Comment by leeroy-travis** [comment]
Commit SHA:0db6eed41c4ca7e4bc9fdc594ed00b4f84c9ff81
No changes to code coverage between the base branch and the head branch

**Comment by leeroy-travis** [comment]
Commit SHA:0db6eed41c4ca7e4bc9fdc594ed00b4f84c9ff81
No changes to code coverage between the base branch and the head branch

File: .github/workflows/publish.yaml [hunk 1]
```diff
@@ -8,6 +8,12 @@ name: Publish
   release:
     types:
       - created
+  # workflow_dispatch is just for testing, and it won't fully work.
+  workflow_dispatch:
+    inputs:
+      name:
+        description: fake version name
+        required: true
```
Comment (skeet70) [inline .github/workflows/publish.yaml:16]: Is this intended to stay in?

File: .github/workflows/publish.yaml [hunk 2]
```diff
@@ -21,39 +27,33 @@ jobs:
       fail-fast: false
       matrix:
         node_version:
-          - 14
           - 16
           - 18
```
Comment (skeet70) [inline .github/workflows/publish.yaml:31]: Should add `20` at the same time if possible, it's going to be LTS-ified soon. To the other node version specifications too.

**Review by skeet70 — APPROVED** [review]

**Comment by skeet70** [comment]
The README also needs to be updated, removing 14 and adding 20.

File: .github/workflows/publish.yaml (hunk 1 above)
Comment (cjyar) [reply .github/workflows/publish.yaml:16]: I guess so. It doesn't do any harm to leave it here, and it'll help when we want to test the workflow in the future.

**Comment by leeroy-travis** [comment]
Commit SHA:2c62e6e6494dcbffcde2beba7d334fa397d7816a
No changes to code coverage between the base branch and the head branch

**Comment by leeroy-travis** [comment]
Commit SHA:2c62e6e6494dcbffcde2beba7d334fa397d7816a
No changes to code coverage between the base branch and the head branch

**Comment by leeroy-travis** [comment]
Commit SHA:b4b0707f44ecc749898fe74eb5061043a40246a0
No changes to code coverage between the base branch and the head branch

**Comment by leeroy-travis** [comment]
Commit SHA:b4b0707f44ecc749898fe74eb5061043a40246a0
No changes to code coverage between the base branch and the head branch

**Comment by leeroy-travis** [comment]
Commit SHA:bbf9af18f634917cd85d09069469472710790c36
No changes to code coverage between the base branch and the head branch

**Comment by leeroy-travis** [comment]
Commit SHA:bbf9af18f634917cd85d09069469472710790c36
No changes to code coverage between the base branch and the head branch

**Pull request merged**